Sync line test cases files

parekh-vivek opened the pull request.

smontoli commented:
Buongiorno,

per procedere con l'analisi della PR, è necessario che venga compilato il form di richiesta accreditamento per l'applicativo, come indicato al punto 4 del paragrafo "Fase 1: processo di accreditamento" del seguente link:

https://github.com/ministero-salute/it-fse-support/blob/8ec92afe117e2459bc04ed5fc897213002f1d649/doc/accreditamento/README.md#fase-1-processo-di-accreditamento

Un saluto,

Il team FSE 2.0

IStacchiotti commented:
Buongiorno, 

siamo in attesa di una commit correttiva per l'aggiornamento della checklist, in seguito alla mail del 14/07 con cui è stato comunicato l'esito negativo delle verifiche preliminari.

Grazie,
Il team FSE 2.0

parekh-vivek commented:
can you please help me to understand what is issue with report-checklist.xlsx file, as yesterday I've updated that file.

IStacchiotti commented:
Buongiorno, 

Non ci risulta un file checklist aggiornato dopo la comunicazione di ieri, neanche sul suo repository locale.
La invitamo gentilmente a riprovare il caricamento tramite commit correttiva.
Se può essere utile la rimandiamo alla documentazione al [link](https://github.com/ministero-salute/it-fse-accreditamento), rimanendo disposizione per eventuali necessità.

La invitiamo anche a pulire il repository, mantenendo solo il file checklist più aggiornato:
<img width="215" height="331" alt="image" src="https://github.com/user-attachments/assets/6f5c8ec5-4b7c-4188-930b-501200b24660" />

Grazie,
Il team FSE 2.0

parekh-vivek commented:
Here I've deleted checklist.xlsx file and updated report-checklist.xlsx file.

<img width="1568" height="283" alt="image" src="https://github.com/user-attachments/assets/06174c03-0c16-44ae-9ad7-6776d1ddb423" />

let me know if still I need to update anything.

parekh-vivek commented:
We have already taken original template and alter the result that we done test 

<img width="1341" height="820" alt="image" src="https://github.com/user-attachments/assets/aaf82999-6246-43ee-a7f3-a0f6d73c835d" />

<img width="1865" height="769" alt="image" src="https://github.com/user-attachments/assets/b7910983-5cc8-4b54-9046-a37588bba623" />

<img width="963" height="200" alt="image" src="https://github.com/user-attachments/assets/57acd3b6-4d65-4234-b1b1-e7aace912956" />

So can you please help me to understand that which changes are not done?

parekh-vivek commented:
Thanks for detail answer, I've modified files and updated again. 
So kindly please review and let me know if I need to change anything. 

Thanks

parekh-vivek commented:
updated with change test cases.

parekh-vivek commented:
updated with change test cases.

parekh-vivek commented:
can you please help me understand why this is not yet visible in the following repository?

🔗 https://github.com/ministero-salute/it-fse-accreditati/tree/main/RESULTS
